prism/src/main/java/org/apache/falcon/resource/proxy/ExtensionManagerProxy.java

```diff
@@ -94,14 +94,16 @@
 
     //SUSPEND CHECKSTYLE CHECK ParameterNumberCheck
     @GET
-    @Path("list/{extension-name}")
+    @Path("list")
     @Produces({MediaType.TEXT_XML, MediaType.APPLICATION_JSON})
     public ExtensionJobList getExtensionJobs(
-            @PathParam("extension-name") String extensionName,
+            @QueryParam("extension-name") String extensionName,
```

Comment:
Understand the intent. Can you see if we can do something like what we do for entity listing. Entity type is part of the path, but, is optional.